backoffice/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "src/main/index.js",
   "scripts": {
     "start": "parcel ./public/index.html -p 8080",
-    "build": "parcel build ./public/index.html"
+    "build": "parcel build public/index.html --public-url ./"
```

Comment:
With this change - the app will work with gh-pages and also in the dev enviorement? wonder if a new build will be required for just gh-pages

Reply:
yes, for the dev enviroment we use the start task, and that generate with the relative paths, and the build task will generate with the current path, basicly we only gonna use the build task to make the gh-pages deploy